provisionersdk/proto/provisioner.proto

```diff
@@ -294,7 +294,6 @@ message Metadata {
     string workspace_owner_login_type =  18;
     repeated Role workspace_owner_rbac_roles = 19;
     bool is_prebuild = 20;
-    string running_workspace_agent_token = 21;
```

Comment:
This was introduced in https://github.com/coder/coder/pull/16951. Howcome we're explicitly removing it here?